modules/op-installing-tekton-hub-without-login-and-rating.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="installing-tekton-hub-without-login-and-rating.adoc_{context}"]
+= Installing {tekton-hub} without login and rating
+
+[role="_abstract"]
+You can install {tekton-hub} on your cluster automatically with default configuration. When using the default configuration, {tekton-hub} does not support login with authorization and ratings for Hub artifacts.
+
+[discrete]
+.Prerequisites
+* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
+
+[discrete]
+.Procedure 
+
+. Create a `TektonHub` CR similar to the following example.
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonHub
+metadata:
+  name: hub
+spec:
+  targetNamespace: openshift-pipelines <1>
+  api:
+    hubConfigUrl: https://raw.githubusercontent.com/tektoncd/hub/main/config.yaml <2>
+    catalogRefreshInterval: 30m     # After every 30min catalog resour
+----
+<1> The namespace in which Tekton Hub must be installed; default is `openshift-pipelines`.
+<2> Substitute with the URL of the `config.yaml` file of your forked repository.
+
+. Apply the `TektonHub` CR.
++
+[source,terminal]
+----
+$ oc apply -f <TektonHub>.yaml <1>
+----
+<1> The file name or path of the `TektonHub` CR.
++
+[NOTE]
+====
+When you apply the `TektonHub` CR, {tekton-hub} is installed on the cluster in the `openshift-pipelines` namespace, with upstream Tekton Catalog content. 
+====
+
+. Check the status of the installation.
++
+[source,terminal]
+----
+$ oc get tektonhub.operator.tekton.dev
+NAME   VERSION   READY   REASON   APIURL                    UIURL
```

Comment:
Output should be in a separate code block from command:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output